etc/nsjail/execute.cfg

```diff
@@ -0,0 +1,137 @@
+name: "compiler explorer compilation/tool sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+max_cpus: 1
+
+log_level: INFO
+
+rlimit_as_type: INF
+rlimit_cpu_type: SOFT
+rlimit_fsize: 1024 # 1GB
+rlimit_nofile: 20
+
+uidmap {
+    inside_id: "0"
+}
+
+gidmap {
+    inside_id: "0"
+}
+
+# must run following as root during system startup
+# cgcreate -a ubuntu:ubuntu -g memory,pids,cpu,net_cls:ce-compile
+cgroup_mem_parent: "ce-compile"
+cgroup_pids_parent: "ce-compile"
+cgroup_net_cls_parent: "ce-compile"
+cgroup_cpu_parent: "ce-compile"
+
+cgroup_mem_max: 1342177280 # 1.25 GiB
+cgroup_pids_max: 4
+cgroup_cpu_ms_per_sec: 800
+
+mount {
```

Comment:
we are probably going to need to be more liberal with mounts here.

`/usr` <- can't think of a reason to not just give compilers all of this
`/bin` <- wouldn't be surprised if some compilers depend on stuff here
`/etc` <- probably some stuff in here that some compilers may want

Reply:
I agree, this was one of the reasons I was having issues getting this to work with cmake and make.

Reply:
Oh got it. I'll happily add those in. Just... seems like a good place to start with little.

Reply:
Reluctant to "just" add `/etc`. Will do something more choice, we were more careful in firejail.

Reply:
probably should have been clearer on `/etc`; I think we are safe to just give access to `/usr` and `/bin` outright, and we will likely need to find things on a case by case basis in `/etc` that are needed and specifically bind mount (or fabricate with `src_content`).

Reply:
Absolutely. So far I've fabricated /etc/passwd only. `ld.so.conf.d,ld.so.conf` are the other two mentioned in the `firejail` config but I don't know if/when/how those are actually used.